docs/pretty-tables.md

```diff
@@ -0,0 +1,142 @@
+# `table.pretty` and changes to printing tables
+
+## Summary
+
+We will add a function to display a pretty version of a table for debugging purposes. We will use this function as the backbone to changing the default implementation of `print(table)` in CLI contexts.
```

Comment:
Wording of "CLI contexts" is misleading, I should remove that. This behavior would be just as useful in something like an in-game console, which is not really CLI.